.github/workflows/twin-validator-test.yaml

```diff
@@ -0,0 +1,21 @@
+name: Twin Validator Test
+
+permissions:
+  issues: write
+  pull-requests: write
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 */3 * * *"
+
+jobs:
+  run-twin-validator-test:
```

Comment:
nit: rename to something like `run-forge-twin-validator-test` so it's easier to grep forge continuous tests :)